bedrock/tabzilla/templates/tabzilla/tabzilla.js

```diff
@@ -559,26 +535,6 @@ var Tabzilla = (function (Tabzilla) {
     // Expose the object for the tests
     Tabzilla.infobar = Infobar;
     var setupGATracking = function () {
```

Comment:
The only thing this function is doing now is adding an unnecessary 500ms delay to submitting the form. Any reason we shouldn't remove this function and all calls to it?